cmake/kokkos_functions.cmake

```diff
@@ -281,7 +281,11 @@ MACRO(kokkos_import_tpl NAME)
       MESSAGE(FATAL_ERROR "Find module succeeded for ${NAME}, but did not produce valid target ${TPL_IMPORTED_NAME}")
     ENDIF()
     IF(NOT TPL_NO_EXPORT)
-      KOKKOS_EXPORT_IMPORTED_TPL(${TPL_IMPORTED_NAME})
+      GET_TARGET_PROPERTY(TPL_ORIGINAL_NAME ${TPL_IMPORTED_NAME} ALIASED_TARGET)
+      IF (NOT TPL_ORIGINAL_NAME)
+        SET(TPL_ORIGINAL_NAME ${TPL_IMPORTED_NAME})
+      ENDIF()
+      KOKKOS_EXPORT_IMPORTED_TPL(${TPL_ORIGINAL_NAME})
```

Comment:
Why is this needed?  Make sure this is in the description of the PR

Reply:
I added a comment. Without this, we are getting
```
-- Performing Test KOKKOS_QUADMATH_COMPILER_SUPPORT
-- Performing Test KOKKOS_QUADMATH_COMPILER_SUPPORT - Success
CMake Error at cmake/kokkos_functions.cmake:187 (INSTALL):
  INSTALL TARGETS given target "Kokkos::LIBQUADMATH" which is an alias.
Call Stack (most recent call first):
  cmake/kokkos_functions.cmake:288 (KOKKOS_EXPORT_IMPORTED_TPL)
  cmake/kokkos_tpls.cmake:86 (KOKKOS_IMPORT_TPL)
  cmake/kokkos_tribits.cmake:247 (INCLUDE)
  CMakeLists.txt:188 (KOKKOS_SETUP_BUILD_ENVIRONMENT)
```